README.md

```diff
@@ -95,6 +95,66 @@ Choose the output format. Currently, `stylish` (default), `json` and `markdown`
 ui5lint --format json
 ```
 
+#### `--ignore-pattern`
+
+Pattern/files that will be ignored for linting. Can be defined also in `ui5lint.config.js`.
+
+**Example:**
+```sh
+ui5lint --ignore-pattern "./test"
+```
+
+## Configuration
+
+UI5 Linter can be easily configured using an external configuration file. This file allows you to customize how the linter behaves, such as ignoring specific files or directories. 
+
+### Configuration File Location
+The configuration file must be placed in the root directory of your project, alongside the `ui5.yaml` and `package.json` files. The linter will automatically detect and load the file when it runs.
+
+### Supported Configuration File Names
+You can name your configuration file in one of the following formats:
+- `ui5lint.config.js`
+- `ui5lint.config.mjs`
+- `ui5lint.config.cjs`
+
+If you need to specify a custom configuration file, you can provide it using the `--config` parameter via the command line.
+
+### Configuration File Format
+
+#### ESM (ECMAScript Modules):
+```js
+export default {
+  ignores: [
+    "test/**/*", 
+    "!test/sap/m/visual/Wizard.spec.js",
```

Comment:
Same here: Rather use an example that is more common for users (which usually have application projects)

Reply:
Now should be better :)